THEME.md

```diff
@@ -205,25 +205,46 @@ It's a good idea to add this to the top of the template to improve the styling o
 
 ## Action Injection
 
-The following will inject the action link (or button) into the e-mail:
+The following will inject the action link (or button) into the e-mail.
 
 ```html
 <!-- Action -->
+<% if(locals.action.constructor !== Array){ %>
+<%    action = [action]; %>
+<% } %>
+
 <% if (locals.action) { %>
-    <p><%- action.instructions %></p>
-    <a href="<%- action.button.link %>" style="color:<%- action.button.color %>" target="_blank">
-        <%- action.button.text %>
-    </a>
+    <% action.forEach(function (actionItem) { -%>
+        <p><%- actionItem.instructions %></p>
+        <a href="<%- actionItem.button.link %>" style="color:<%- actionItem.button.color %>" target="_blank">
+            <%- actionItem.button.text %>
+        </a>
+    <% }) -%>
 <% } %>
 ```
 
 It's a good idea to add this to the top of the template to specify a fallback color for the action button in case it wasn't provided by the user:
 
 ```html
 <%
+if (locals.action){
+
```

Comment:
Please omit the extra line breaks(s) here and in the theme files, as well as fix the code style (spacing) so it matches the project code style.